build.gradle.kts

```diff
@@ -20,20 +20,9 @@ dependencies {
     compileOnly("org.projectlombok:lombok:1.18.34")
     implementation("org.projectlombok:lombok:1.18.34")
 
-    implementation(fileTree(mapOf("dir" to "libs", "include" to listOf("*.jar"))))
+    implementation("org.teavm:teavm-jso-apis:0.10.0")
```

Comment:
are both of these required separately?

Reply:
seems like teavm-jso-apis includes teavm-jso